ecommerce/src/main/kotlin/io/hhplus/ecommerce/product/application/ProductStatisticsSyncScheduler.kt

```
@@ -0,0 +1,127 @@
+    }
+
+    /**
+     * Redis 키에서 productId 추출
+     *
+     * @param key Redis 키 (예: "product:view:123")
+     * @return 추출된 productId (실패 시 null)
+     */
+    private fun extractProductId(key: String): Long? {
+        return try {
+            key.substringAfterLast(":").toLongOrNull()
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)